ansible/playbooks/tools/upgrade-reboot.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+# upgrade - upgrade packages which are safe to be updated.
+# This means no packages will be removed.
+# The machine is rebooted if necessary. Skip the reboot
+# tag to avoid that.
+# Copyright (C) 2020 Ellen Papsch <ellenpapsch@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-only
+
+- hosts: debops_all_hosts
+  become: true
+  tasks:
+
+    - name: Upgrade safe packages with refreshed cache
+      apt:
+        update_cache: yes
+        upgrade: safe
```

Comment:
You can add a `when: ansible_pkg_mgr == 'apt'` condition here, then we could expand the playbook for more OSes if somebody is interested. However that won't work without fact gathering. Your choice.

The `safe` string should be quoted.

Reply:
I refrained from `when: ansible_pkg_mgr == 'apt'`. I guess if someones wants to use the playbook with Fedora (or whatever) they also want the rest of debops to work with it and that would be quite an undertaking.